hivemq-edge/src/frontend/src/modules/Subscriptions/AdapterSubscriptionManager.spec.cy.tsx

```diff
@@ -0,0 +1,100 @@
+/// <reference types="cypress" />
+
+import AdapterSubscriptionManager from '@/modules/Subscriptions/AdapterSubscriptionManager.tsx'
+import { Route, Routes } from 'react-router-dom'
+import { Node } from 'reactflow'
+
+import { ReactFlowTesting } from '@/__test-utils__/react-flow/ReactFlowTesting.tsx'
+import { MOCK_NODE_ADAPTER } from '@/__test-utils__/react-flow/nodes.ts'
+import useWorkspaceStore from '@/modules/Workspace/hooks/useWorkspaceStore.ts'
+import { mockAdapter, mockProtocolAdapter } from '@/api/hooks/useProtocolAdapters/__handlers__'
+
+const getWrapperWith = (n?: Node[]) => {
+  const Wrapper: React.JSXElementConstructor<{ children: React.ReactNode }> = ({ children }) => {
+    const { nodes } = useWorkspaceStore()
+    return (
+      <ReactFlowTesting
+        config={{
+          initialState: {
+            nodes: n,
```

Comment:
following the previous comment of using `nodes` rather than `n`, you'd be able to just write `nodes,` here